cockatrice/src/gameselector.cpp

```diff
@@ -290,13 +292,15 @@ void GameSelector::actSelectedGameChanged(const QModelIndex &current, const QMod
     joinButton->setEnabled(game.player_count() < game.max_players() || overrideRestrictions);
 }
 
-void GameSelector::setAlteredFiltersText(const int numAlteredFilters)
+void GameSelector::setFilteredGamesLabel()
 {
-    if (alteredFiltersLabel != nullptr) {
-        if (numAlteredFilters == 0) {
-            alteredFiltersLabel->setText(tr("Default filters applied"));
-        } else {
-            alteredFiltersLabel->setText(tr("%1 filter(s) applied").arg(numAlteredFilters));
-        }
+    if (filteredGamesLabel != nullptr && gameListProxyModel != nullptr &&
```

Comment:
```suggestion
    if (filteredGamesLabel && gameListProxyModel && gameListModel)
```

How often would one of these conditions fail?

Reply:
Oh fair point - everything is set in the constructor; I think I can safely remove this check.